src/core_plugins/kibana/public/dashboard/components/exit_full_screen_button.js

```diff
@@ -64,9 +64,10 @@ export class ExitFullScreenButton extends PureComponent {
             aria-label="Exit full screen mode"
             className="exitFullScreenMode"
             onClick={this.props.onExitFullScreenMode}
+            data-test-subj="exitFullScreenModeButton"
```

Comment:
It seemed more efficient and analogous to user behavior to click the button itself instead of one of its children.